igor_src/ExtractFeatures.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * File:   ExtractFeatures.h
+ *
+ *      Author: Carlos Olivares
+ *
+ *  This source code is distributed as part of the IGoR software.
+ *  IGoR (Inference and Generation of Repertoires) is a versatile software to analyze and model immune receptors
+ *  generation, selection, mutation and all other processes.
+ *   Copyright (C) 2017  Quentin Marcou
```

Comment:
Not sure how this works but I guess you can put up: Copyright (C) 2017-  Quentin Marcou 2019 - Carlos Olivares
something along those lines